utilities/blob_db/blob_db_impl.cc

```diff
@@ -2244,6 +2221,39 @@ Iterator* BlobDBImpl::NewIterator(const ReadOptions& opts,
                             column_family, this);
 }
 
+Status DestroyBlobDB(const std::string& dbname, const Options& options,
+                     const BlobDBOptions& bdb_options) {
+  const ImmutableDBOptions soptions(SanitizeOptions(dbname, options));
+  Env* env = soptions.env;
+
+  Status status;
+  std::string blobdir;
+  blobdir = (bdb_options.path_relative) ? dbname + "/" + bdb_options.blob_dir
+                                        : bdb_options.blob_dir;
+
+  std::vector<std::string> filenames;
+  env->GetChildren(blobdir, &filenames);
+
+  for (const auto& f : filenames) {
+    uint64_t number;
+    FileType type;
+    if (ParseFileName(f, &number, &type) && type == kBlobFile) {
+      Status del = env->DeleteFile(blobdir + "/" + f);
+      if (status.ok() && !del.ok()) {
+        status = del;
```

Comment:
You would be discarding all subsequent non-ok statuses here, isn't it? Do you need to somehow handle them as well? 
Example: Let say `filenames` vector is of size 5. Each  `DeleteFile` failed with a different status/different reason. You enter line 2243 only for setting the first failure status, and never re-enter it again as status.ok() is false from here on. So, what is the plan for handling the subsequent non-ok statuses?